docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,43 @@
+# Contribution Guidelines
+
+When contributing to this repository, please [submit a new issue](https://github.com/formatc1702/WireViz/issues) first to discuss the proposed change, before submitting a pull request.
+
+## Submitting a new Issue
+
+- First, search existing (open and closed) issues for any related issues.
+  - You might then find an existing answer or suggested solution to your issue, possibly also an existing PR you can test.
+  - When finding existing issues that seem related to your issue, please include references (# followed by issue number) to related issues in your new issue description, or if a very similar issue is still open, consider adding a comment in that issue instead of creating a new one.
+- When appropriate, please prefix your issue title with one of these category prefixes followed by a space:
+  - **[bug]** When the issue seems to be caused by a bug.
+  - **[feature]** When requesting a feature change or new feature.
+  - **[internal]** When suggesting code improvements that doesn't change any output.
+  - **[doc]** For documentation issues.
+  - **[meta]** For issues about the development or contribution process.
```

Comment:
We have lately also used some other categories, not listed here, but I assume this doesn't have to be a complete list... no changes required.

Reply:
Yep. This list should cover most cases. Anyway it's not a super strict rule, but it helps reduce the chaos a bit.